app/models/chunk.py

```diff
@@ -0,0 +1,18 @@
+from app import db
+
+
+class Chunk(db.Model):
+    """
+    Chunk is a minimal item to translate.  Translation
+    consists of multiple chunks.
+    """
+    __tablename__ = 'chunk'
+
+    chunk_id = db.Column(db.Integer, primary_key=True)
+    text = db.Column(db.Text)
+    translation_id = db.Column(db.Integer,
+                               db.ForeignKey('translation.translation_id'))
+
+    def __init__(self, text, translation_id):
```

Comment:
You don't need to add this `__init__` here, SQLAlchemy's `Model` already has one